x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -110,21 +124,39 @@ export class HeadlessChromiumDriverFactory {
     private logger: Logger,
     private binaryPath: string,
     private basePath: string
-  ) {
-    const dataDir = getDataPath();
-    fs.mkdirSync(dataDir, { recursive: true });
-    this.userDataDir = fs.mkdtempSync(path.join(dataDir, 'chromium-'));
```

Comment:
I'm not sure why the userDataDir was being created in the instantiation of this class. It seems problematic since this class is instantiated in plugin setup.